src/utilities/process-readme.js

```diff
@@ -1,6 +1,70 @@
 const url = require('url');
 
-const beginsWithDocsDomainRegex = /^https?:\/\/webpack\.js\.org/;
+const beginsWithDocsDomainRegex = /^(?:https?:)\/\/webpack\.js\.org/;
+const inlineLinkRegex = /\[[^\]]*\]\(([^)]+)\)/g;
+const externalLinkRegex = /^[ \t]*\[[^\]]+\]:[ \t]+([^ \n]+)/gm;
+
+const fragmentLinkMap = {
```

Comment:
Can this include the faulty readme file source so we know where to put a PR to fix it?

Reply:
The link map itself is source agnostic. It's just a mapping. The source will be highlighted in whatever code uses the map